client/src/main/java/io/confluent/kafka/schemaregistry/client/rest/RestService.java

```diff
@@ -1705,6 +1930,16 @@ public void setHttpHeaders(Map<String, String> httpHeaders) {
 
   public void setProxy(String proxyHost, int proxyPort) {
     this.proxy = new Proxy(Proxy.Type.HTTP, new InetSocketAddress(proxyHost, proxyPort));
+    if (this.useApacheHttpClient){
+      try {
+        URI uri = new URI(proxyHost);
+        proxyHost = uri.getHost();
+      } catch (URISyntaxException e) {
+        proxyHost = proxyHost.replaceFirst("^https?://", "");
```

Comment:
I'm not sure why having an "http://" or "https://" would cause a URISyntaxException